release-calendar/src/entities/release.ts

```diff
@@ -14,17 +14,30 @@
  * limitations under the License.
  */
 
-import {Entity, PrimaryColumn} from 'typeorm';
+import {Entity, PrimaryColumn, Column} from 'typeorm';
+import {Channel} from '../types/channel';
```

Comment:
There's a way that TypeScript can already know your custom defined types, so you don't have to import it anywhere. Importing it is okay for now, but let's figure this out in a future PR.